src/state/index.ts

```diff
@@ -19,4 +19,27 @@ export const reducers = (history: History<any>): { [key in keyof AppState]: Redu
 
 export const history = createBrowserHistory({ basename: 'localsurvey' });
 
-export const store = createStore(combineReducers(reducers(history)), composeWithDevTools());
+// Define a preloadedState for createStore()
+// See: https://redux.js.org/recipes/structuring-reducers/initializing-state
+const preloadedState: any = {
+    homepageState: {
+        surveys: [
+            {
+                key: '1',
+                name: 'Zoo visit questionnaire',
+                createdAt: 1549238929,
```

Comment:
Wdyt about using moment for this?

Reply:
Am using native JS right now in the [render method for the displaying table](https://github.com/decrn/localsurvey/pull/23/files#diff-01b0d7ec902091757b45cdca64746cf6R25).

Open for suggestions. Will need to support D/T better at some point. Might as well be now.

Reply:
@tuur29

Reply:
We could use moment for this, but to me it seems unnecessary. For one, the list is displayed as a table which normally contains real data instead of something fuzzy. Secondly, I don't see another place where we could use moment, so we would be adding a dependency for just this one instance.

Reply:
Let's leave the data as timestamp, if necessary we can always map to `Moment` objects when reading from state